gen2-camera-demo/main.py

```diff
@@ -41,7 +41,10 @@
 pcl_converter = None
 if point_cloud:
     if out_rectified:
-        from projector_3d import PointCloudVisualizer
+        try:
+            from projector_3d import PointCloudVisualizer
+        except ImportError as e:
+            raise ImportError(f"\033[1;5;31mError occured when importing PCL projector: {e}. Try disabling the point cloud \033[0m ")
```

Comment:
User cannot disable pcl since it's hardcoded as true, we need to add command line params also

Reply:
Thanks. Added `-pcl` arg to disable it.

Reply:
Aahh right.